config/migrations/cockroachdb/5_constraints_with_description.up.sql

```diff
@@ -0,0 +1,2 @@
+-- Add description column to constraints
+ALTER TABLE constraints ADD COLUMN description TEXT;
```

Comment:
I think we wanna have a `NOT NULL` constraint like

`description TEXT NOT NULL`

we do that elsewhere for all the descriptions: https://github.com/flipt-io/flipt/blob/main/config/migrations/sqlite3/0_initial.up.sql#L13

even though they are technically optional from the API standpoint, this helps us at the storage layer so that we don't have to check for `nil`

also it might be good to add some tests at both the storage and API layers to make sure `description` is indeed optional if you don't have them already

Reply:
@markphelps Cool I've added the `NOT NULL` constraint but it seems like it needs a default value in order for the migration to work, which I guess makes sense.

I've also kept some tests the same to observe the backwards compatibility of this change, along with providing the `Description` field on `UpdateConstraint` to observe that the description does indeed get updated.

Reply:
i dont think we want a default, that would mean its not actually optional.

I think it would insert the empty string by default